integration/flink/src/test/java/io/openlineage/flink/FlinkContainerUtils.java

```diff
@@ -0,0 +1,188 @@
+package io.openlineage.flink;
+
+import com.google.common.io.Resources;
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+import java.util.Arrays;
+import java.util.List;
+import lombok.SneakyThrows;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.MockServerContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.lifecycle.Startable;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.MountableFile;
+
+public class FlinkContainerUtils {
+
+  static MockServerContainer makeMockServerContainer(Network network) {
+    return new MockServerContainer(
+            DockerImageName.parse("jamesdbloom/mockserver:mockserver-5.12.0"))
+        .withLogConsumer(of -> consumeOutput("mockserver", of))
+        .withNetwork(network)
+        .withNetworkAliases("openlineageclient");
+  }
+
+  static GenericContainer<?> makeSchemaRegistryContainer(Network network, Startable startable) {
+    return genericContainer(network, "confluentinc/cp-schema-registry:6.2.1", "schema-registry")
+        .withExposedPorts(28081)
+        .withEnv("SCHEMA_REGISTRY_KAFKASTORE_BOOTSTRAP_SERVERS", "PLAINTEXT://kafka:9092")
+        .withEnv("SCHEMA_REGISTRY_HOST_NAME", "schema-registry")
+        .withEnv("SCHEMA_REGISTRY_LISTENERS", "http://schema-registry:8081,http://0.0.0.0:28081")
+        .dependsOn(startable);
+  }
+
+  static GenericContainer<?> makeKafkaContainer(Network network, Startable zookeeper) {
+    return genericContainer(network, "confluentinc/cp-kafka:6.2.1", "kafka")
```

Comment:
Can we use other Kafka image, such as `wurstmeister/kafka:2.13-2.8.1`? Confluent images don't work on ARM.

Reply:
please, check it on ARM then ;-)